### PR TITLE
Resolve HttpClient Timeout Error On Max Retries

### PR DESCRIPTION
**Pull request by madhukar-agsft**

### CODE REVIEW

File: src/SimulationCSharpClient/Client/HttpRetryMessageHandler.cs [hunk 1]
```diff
@@ -9,18 +9,25 @@ namespace SimulationCSharpClient.Client
...
         public HttpRetryMessageHandler(int maxRetries = 6, HttpClientHandler handler = null)
             : base(handler ?? new HttpClientHandler())
         {
-            this.maxRetries = (maxRetries <= 0 || maxRetries > 6) ? 6 : maxRetries;
+            this.MaxRetries = (maxRetries <= 0 || maxRetries > 6) ? 6 : maxRetries;
+        }
+
+        public int MaxRetries { get; private set; }
+
+        public double TotalWaitDurationInSecs
```
Comment (chadqueen) [inline src/SimulationCSharpClient/Client/HttpRetryMessageHandler.cs]: reference all comments on workflow client PR

**Review by chadqueen — CHANGES_REQUESTED** [review]
reference all comments on workflow client PR

File: src/SimulationCSharpClient/Client/HttpRetryMessageHandler.cs (hunk 1 above)
Comment (madhukar-agsft) [reply src/SimulationCSharpClient/Client/HttpRetryMessageHandler.cs]: @chadqueen Modified as per your comments on the Workflow-csharp-client PR

**Comment by madhukar-agsft** [comment]
> reference all comments on workflow client PR

@chadqueen Modified as per your comments on the Workflow-csharp-client PR

**Review by alexDrinkwater — APPROVED** [review]

**Pull request merged**